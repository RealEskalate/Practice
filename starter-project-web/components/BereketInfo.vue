<template>
    <v-card color="white">
          <v-card-title class="black--text">
            <h2>About me</h2>
          </v-card-title>
          <v-list-item class="black--text">
            <v-list-item-content>
              <v-list-item-title>
                Bereket Heramo
              </v-list-item-title>
              <v-flex offset-md-4>
                <v-list-item-title class="mt-1">
                  Lorem ipsum dolor sit amet, consectetur adipiscing elit,<br/> 
                  sed do eiusmod tempor incididunt ut labore et dolore magna aliqua
                </v-list-item-title>
              </v-flex>
              <v-list-item-title>
                
                <img
                  class="ml-3"
                  width="150px"
                  src="../assets/bereket.jpg"
                />
              

              </v-list-item-title>
              
              
              
            </v-list-item-content>
          </v-list-item>
        </v-card>
</template>

<script>
export default {
name:'BereketInfo'
}
</script>

<style>

</style>
