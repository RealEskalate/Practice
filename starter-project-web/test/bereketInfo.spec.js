import { mount, createLocalVue } from '@vue/test-utils'
import BereketInfo from '../components/BereketInfo.vue'
import Vuetify from 'vuetify'

const localVue = createLocalVue()




describe('Bereket info renders', () => {
  let vuetify
  beforeEach(() => {
    vuetify = new Vuetify()
  })

  test('renders properly', () => {
    const wrapper = mount(BereketInfo, { vuetify, localVue })
    expect(wrapper.text()).toContain('Bereket Heramo')
    expect(wrapper.html()).toMatchSnapshot()
  })
})

