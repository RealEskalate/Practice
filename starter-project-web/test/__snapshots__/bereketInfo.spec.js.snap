// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bereket info renders renders properly 1`] = `
"<v-card color=\\"white\\">
  <v-card-title class=\\"black--text\\">
    <h2>About me</h2>
  </v-card-title>
  <v-list-item class=\\"black--text\\">
    <v-list-item-content>
      <v-list-item-title>
        Bereket Heramo
      </v-list-item-title>
      <v-flex offset-md-4=\\"\\">
        <v-list-item-title class=\\"mt-1\\">
          Lorem ipsum dolor sit amet, consectetur adipiscing elit,<br>
          sed do eiusmod tempor incididunt ut labore et dolore magna aliqua
        </v-list-item-title>
      </v-flex>
      <v-list-item-title><img width=\\"150px\\" src=\\"../assets/bereket.jpg\\" class=\\"ml-3\\"></v-list-item-title>
    </v-list-item-content>
  </v-list-item>
</v-card>"
`;
